src/stores/Annotation/Annotation.js

```diff
@@ -677,14 +677,22 @@ export const Annotation = types
       self.setDraftSelected();
       self.versions.draft = result;
       self.setDraftSaving(true);
-
-      return self.store.submitDraft(self, params).then(self.onDraftSaved);
+      return self.store.submitDraft(self, params).then((res) => {
+        self.onDraftSaved(res);
```

Comment:
Does any other callers of saveDraft require that the onDraftSaved callback be the return of the function promise? I noticed we are changing to use the raw result instead so that we can get access to this information, I'm wondering if that might break other usages. Do we know for certain the usages expectation of this return?

Reply:
I'm going to approve this with the caveat that we may have to consider this depending on whether any integration tests fail (and subsequently answer this question for us 😅).